bokun.yaml

```diff
@@ -5537,413 +5734,535 @@ definitions:
-        description: 'The photo selected for use as the key photo for the product in lists and overviews.'
-      keywords:
-        type: array
-        description: 'A list of keywords describing the product.'
-        items:
-          type: string
-      languages:
-        type: array
-        items:
-          type: string
-        description: 'A list of languages which the product content is available in.'
-      lastPublished:
-        type: string
-        description: 'The date that the product was last changed and published.'
-      creationDate:
-        type: string
-        description: 'The date the product was first created.'
-      lastModified:
-        type: string
-        description: 'The date that the product content was last modified.'
-      locationCode:
-        $ref: '#/definitions/LocationCodeDto'
-        description: 'The city that represents the main location for this activity. Specified as a location from the United Nations Code for Trade and Transport Locations (UN/LOCODE).'
+        description: 'Specifies activity duration in human language and formated to the locale specified in the profile.'
       minAge:
         type: integer
         format: int32
-        description: 'Represents the age limit for the activity, if any.'
+        description: 'Represents the age lower limit for the activity, if any.'
       nextDefaultPrice:
         type: number
-        description: 'The price for 1 person in the default category, assuming the future departure date nearest in time to the current time.'
-      noPickupMsg:
-        type: string
-      photos:
-        type: array
-        description: 'A list of photos showcasing this product.'
-        items:
-          $ref: '#/definitions/PhotoDto'
+        description: 'The price for 1 person in the default category, assuming the future departure date nearest in time to the current time. Amount in default locale currency.'
+      nextDefaultPriceMoney:
+        type: object
+        properties:
+          amount:
+            type: number
+          currency:
+            type: string
+        description: 'The price for 1 person in the default category, assuming the future departure date nearest in time to the current time. Amount and currency in default locale currency.'
+      pickupService:
+        type: boolean
+        description: 'Deprecated. Pick up service is now specified in meetingType.'
       pickupAllotment:
         type: boolean
         description: 'Whether pick up has separate capacity from the activity (for example, activity can take 60 pax but the pick up bus can only take 30).'
+      pickupAllotmentType:
+        type: string
+        enum:
+          - NO_ALLOTMENT
+          - USE_ALLOTMENT
+          - USE_COMBO_ALLOTMENT
+        description: 'Not working. Specifies pick up allotment type.'
+      useComponentPickupAllotments:
+        type: boolean
+        description: 'Not working. True if pickupAllotmentType == USE_COMBO_ALLOTMENT.'
```

Comment:
Probably best to exclude "not working" and file a bug or something like that.

Reply:
will review with Tadas and file as a bug

Reply:
Recommended: File this one as bug